nni/algorithms/compression/v2/pytorch/pruning/auto_compress_pruner.py

```diff
@@ -46,7 +46,9 @@ def _iterative_pruner_reset(self, model: Module, config_list: List[Dict] = [], m
     def allocate_sparsity(self, new_config_list: List[Dict], model: Module, masks: Dict[str, Dict[str, Tensor]]):
         self._iterative_pruner_reset(model, new_config_list, masks)
         self.iterative_pruner.compress()
-        _, _, _, _, config_list = self.iterative_pruner.get_best_result()
+        best_result = self.iterative_pruner.get_best_result()
+        assert best_result is not None, 'Best result does not exist, iterative pruner may not start pruning.'
+        _, _, _, _, config_list = best_result
```

Comment:
Irrelevant to this PR, but this looks really strange to me...

Reply:
agree, will change these in future